zerver/lib/markdown/__init__.py

```diff
@@ -1073,6 +1073,18 @@ def handle_tweet_inlining(
         if info['remove'] is not None:
             info['parent'].remove(info['remove'])
 
+    def handle_youtube_url_inlining(
+        self,
+        root: Element,
+        found_url: ResultWithFamily[Tuple[str, Optional[str]]],
+        youtube: str,
+        url: str,
```

Comment:
The `url` parameter can be extracted from `found_url` which would make this function signature cleaner.

```py
(url, text) = found_url.result
yt_id = self.youtube_id(url)
```

Reply:
Got it. I'll make the change.